pkg/controller/constraint/constraint_controller.go

```diff
@@ -317,14 +321,28 @@ func (r *ReconcileConstraint) Reconcile(request reconcile.Request) (reconcile.Re
 }
 
 func (r *ReconcileConstraint) defaultGetPod() (*corev1.Pod, error) {
```

Comment:
can this be reused between the 2 controllers without copy-pasting it ?

Reply:
Unified the two functions, PTAL.